IFC4x3/Constants/s/SOIL_BORING_POINT_3R7zb_0j98qwjdYOilbWE4.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SOIL_BORING_POINT_3R7zb_0j98qwjdYOilbWE4" Name="SOIL_BORING_POINT" UniqueId="db1fd97e-02d2-48d3-ab67-898b2f960384" Status="Deprecated" />
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SOIL_BORING_POINT_3R7zb_0j98qwjdYOilbWE4" Name="SOIL_BORING_POINT" UniqueId="db1fd97e-02d2-48d3-ab67-898b2f960384" />
 
```

Comment:
Distinction between this and an IfcBorehole?

Reply:
Same as above. 😄